generators/sram-bist/src/test/scala/BistTopSpec.scala

```diff
@@ -738,7 +736,387 @@ class BistTopSpec extends AnyFlatSpec with ChiselScalatestTester {
       testhelpers.c.io.bistSignature.expect(misrModelB.state.U)
     }
   }
+  it should "work with chiseltest SRAMs" in {
```

Comment:
I split this test into the four earlier tests, so if you could potentially incorporate this into the other tests that would be great (should just be a matter of modifying BistTopTestHelpers.testBistMethodFull). Delete this test after you've done that.